private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -214,3 +221,19 @@ func inverseEncoding(encoding bufconvert.MessageEncoding) (bufconvert.MessageEnc
 		return 0, fmt.Errorf("unknown message encoding %v", encoding)
 	}
 }
+
+// wktToPath returns the import path of the proto file for a well known type.
+func wktToPath(fulltype string) string {
+	msgType, err := protoregistry.GlobalTypes.FindMessageByName(protoreflect.FullName(fulltype))
```

Comment:
I'm skeptical of this approach We should never be using the global registry for anything we do here. The global registry changes depending on what happens to be linked into the binary.

Reply:
I had suggested this as an alternative to a hand-written hard-coded map of type names to file names. The limitation you mention is easily worked around using underscore imports.

But another option would be to have a code generator (one that is run alongside `storage-to-go`) generate that hard-coded index by actually compiling the files in `datawkt` first and examining all of the types therein. That way it is not hand-coded and will never be inaccurate/stale.